classes.js

```diff
@@ -0,0 +1,256 @@
+// # Class Exercises
+
+
+// (1 - 8 are the same as on the previous lab, but creating classes using ES6 syntax instead of constructor functions)
+
+// ## Question 1
+
+// a. Write a class called `Person` that has 3 properties: a first name, a last name and a middle name. Create 2 instances of a `Person`. Print one of their first names.
+
+class Person {
+    constructor(first, last, middle) {
+        this.first = first;
+        this.last = last; 
+        this.middle = middle;
+    } 
+    fullName() {
+        return `${this.first} ${this.middle} ${this.last}`
+    }
+}
+let bob = new Person ("bob", "robert", "bobby");
+let sue = new Person ("sue", "soo", "su");
+// console.log(sue.first)
+
+
+// b. Write a method in `Person` called `fullName` that will return a formatted string of an instance's full name. Call this method on both the instances you created in part a.
+
+// console.log(bob.fullName())
+// console.log(sue.fullName())
+
+
+
+// ## Question 2
+
+// a. Create a class called `Book` that has properties `title`, `author` and `rating`. Create some instances of `Book`.
+
+class Book {
+    constructor(title, author, rating) {
+        this.title = title;
+        this.author = author;
+        this.rating = rating; 
+    }
+    isGood() {
+        if(this.rating >= 7) {
+            return true
+        } else {
+            return false
+        }
+    }
+}
+
+let books = new Book("Half of a Yellow Sun", "Chimamanda Adichie", 9) 
+let bs = new Book("Trump: The Art of the Deal", "Donald Trump", 1.9)
+
+// console.log(books)
+
+// b. Add a method to `Book` called `isGood` that returns `true` if its rating is greater than or equal to 7
+
+// console.log(bs.isGood())
+
+// ## Question 3
+
+// a. Create a `Dog` class with four properties: `name (string), breed (string), mood (string), and hungry (boolean)`.
+
+class Dog {
+    constructor(name, breed, mood, hungry) {
+        this.name = name;
+        this.breed = breed; 
+        this.mood = mood;
+        this.hungry = hungry
+    }
+    playFetch() {
+        // console.log("Roof!")
+        return this.hungry = true, this.mood = "playful"
+    }
+    feed() {
+        if(this.hungry = false) {
```

Comment:
Here you're setting this.hungry to false instead of comparing.

Remember one equal sign is to set and three is to compare.